modules/cas_disk/exp_obj.c

```diff
@@ -575,8 +568,6 @@ int casdsk_exp_obj_create(struct casdsk_disk *dsk, const char *dev_name,
 
 error_set_geometry:
 	blk_cleanup_queue(exp_obj->queue);
-error_init_queue:
-	blk_mq_free_tag_set(&dsk->tag_set);
 error_init_tag_set:
 	_casdsk_exp_obj_clear_dev_t(dsk);
 error_dev_t:
```

Comment:
As you reordered the operations, you should also adjust error handling to the changes. Additionally, I'm not sure if we can simply get rid of `blk_mq_free_tag_set()` as `_casdsk_init_tag_set()` is still present in the code and as far as I see it wasn't affected by changes in the kernel.